### ISSUE TITLE
home-assistant complains of missing `bluetoothctl`

### ISSUE THREAD

**Issue by lilyball** [NixOS/nixpkgs#211879]
### Describe the bug
I have `home-assistant` configured via the NixOS module with the `bluetooth` component enabled. When I check the logs of Home Assistant, it has a warning log that looks like

```
Logger: root
Source: components/bluetooth/wrappers.py:250 
First occurred: 1:19:17 AM (1 occurrences) 
Last logged: 1:19:17 AM

Could not determine BlueZ version, bluetoothctl not available, assuming 5.51+
```

### Steps To Reproduce
1. Configure `services.home-assistant`. Config should at least contain
   ```nix
   {
     services.home-assistant = {
       enable = true;
       config = {
         bluetooth = {};
       };
     };
   }
   ```

### Expected behavior
There should be no bluetooth-related warnings

### Additional context
I noticed this when trying to figure out why bluetooth stopped working in my Home Assistant install. I ended up rebooting the whole machine which fixed it. I don't know if giving Home Assistant access to `bluetoothctl` would have changed anything, but at the very least I shouldn't be getting a warning about it.

### Notify maintainers
@fabaff @globin @mweinelt @Mic92

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
❯ nix run nixpkgs#nix-info -- -m
 - system: `"aarch64-linux"`
 - host os: `Linux 5.15.86, NixOS, 22.11 (Raccoon), 22.11.20230106.2dea899`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.11.1`
 - channels(root): `"nixos, nixos-hardware"`
 - nixpkgs: `/nix/store/jkl0nk32sz83k6dm15lm8hqv2ack463w-source`
```

home-assistant itself is patched in from nixos-unstable

**Comment by mweinelt** [comment]
Originates here: https://github.com/hbldh/bleak/blob/v0.19.5/bleak/backends/bluezdbus/version.py#L14

**Comment by mweinelt** [comment]
Did you manage to solve this issue in the meantime?

**Comment by lilyball** [comment]
I do have `hardware.bluetooth.enable` turned on, and Home Assistant is still complaining that it can't determine the BlueZ version.

`hardware.bluetooth.enable` adds bluez to the environment packages, but it's unclear to me if Home Assistant runs in a context that gives it access to the system-installed packages. I'm not particularly familiar with systemd services, but I know the macOS equivalent, launchd agents/services, do not run any sort of shell initialization and therefore only have the default PATH available (e.g. `/usr/bin:/bin`) unless otherwise configured. So I'm wondering if Home Assistant needs to add bluez to its PATH manually when the bluetooth module is enabled.

**Comment by mweinelt** [comment]
Can you try the following? 

```nix
systemd.services.home-assistant.path = with pkgs; [ bluez ];
```

**Comment by mweinelt** [comment]
Closing for lack of feedback. Feel free to reopen when you have something to share.

**Comment by mweinelt** [comment]
Possibly fixed in 21317d93e1f739222c4781dcd5ff520337f72bef
